server/pom.xml

```diff
@@ -185,6 +185,11 @@
             <groupId>it.unimi.dsi</groupId>
             <artifactId>fastutil</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.ben-manes.caffeine</groupId>
+            <artifactId>caffeine</artifactId>
+            <version>2.3.1</version>
```

Comment:
Version should not be specified here, it should come in through dependencyManagement from the main pom.

Reply:
Fixed